Gems/Gestures/Code/CMakeLists.txt

```diff
@@ -43,6 +43,7 @@ ly_add_target(
 # All types of applications use the same module.
 ly_create_alias(NAME Gestures.Clients  NAMESPACE Gem TARGETS Gestures)
 ly_create_alias(NAME Gestures.Servers  NAMESPACE Gem TARGETS Gestures)
+ly_create_alias(NAME Gestures.Unified  NAMESPACE Gem TARGETS Gestures)
```

Comment:
Seems odd to me that Gestures Gem would compile for servers - isn't it for mobile gesture input?

Reply:
Yes, changes like this will be saved for a future change.